app/models/concerns/push_updates_for_subscription.rb

```diff
@@ -39,4 +29,26 @@ def app_name
     end
   end
 
+  def data_to_send
+    data = { category: 'message',
+             message: message_body,
+             is_private: message.is_private,
+             item_id: item_id,
+             author_id: message.sender_id,
+             message_id: message.id }
+    data[identity] = message.related_object.id
```

Comment:
Do we really need to open up the related_object just to get the id? Can't we use `messageable_id` ?

```suggestion
    data[messageable_id] = message.messageable_id
```

I don't feel it's a good idea to introduce another column name `identity`. Instead, send `messageable_id` and `messageable_type`